dashboard/utils.py

```diff
@@ -179,6 +212,12 @@ def has_passed_course(self, course_id):
         Returns:
             bool: whether the user has passed the course
         """
+        version = settings.FEATURES.get("FINAL_GRADE_ALGO", "v0")
```

Comment:
Can you use a more meaningful variable name than `version`? Also, please use the name `FINAL_GRADE_ALGORITHM` instead of `FINAL_GRADE_ALGO`: it's clearer.